### ISSUE TITLE
SaveWindowPosition shrinks the window size if scaling is enabled in system settings.

### ISSUE THREAD

**Issue by benwwchen** [MahApps/MahApps.Metro#4417]
## Describe the bug

I found an incorrect behavior regarding SaveWindowPosition on a PC with a scaling enabled display.

My application can save and restore window position **correctly** on other machines **without scaling enabled** in system display settings. However, on the PC **with scaling enabled**, the window **shrinks** a lot every time it reopens. I'm guessing it's because the implementation of SaveWindowPosition is not taking the scaling factor into account so it restores the window to an incorrect state.

Thanks for the great work by the way and thanks for your attention.

## Steps to reproduce

On a Windows 11 PC with Scale setting in System - Display set to 150%,
1. Close the window
2. Reopen the window
3. The window is restored to about 2/3 of the size before closing
4. Repeat step 1 and 2, the window will become smaller every time it reopens.

## Expected behavior

The window restores to the correct size when it reopens.

## Actual behavior

The window restores to a smaller size when it reopens every time.

## Environment

```
MahApps.Metro version: v3.0.0
Windows build number: Windows 11 Pro 21H2 [OS Build 22000.1042]
Visual Studio: 2022
Target Framework: .Net Framework v4.8
```

**Comment by benwwchen** [comment]
Just found a PR #4407 regarding the same issue after posting😂. Will it get merged?

**Comment by punker76** [comment]
Fixed with #4407
